var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -44,12 +49,24 @@ class Qt(Package):
             description="Build the Webkit extension")
     variant('examples',   default=False,
             description="Build examples.")
+    variant('framework',   default=False,
+            description="Build as a macOS Framework package.")
+    variant('tools',      default=True,
+            description="Build tools, including Qt Designer.")
     variant('dbus',       default=False,
             description="Build with D-Bus support.")
     variant('phonon',     default=False,
             description="Build with phonon support.")
     variant('opengl',     default=False,
             description="Build with OpenGL support.")
+    variant('sql',        default=True,
+            description="Build with SQL support.")
+    variant('shared',     default=True,
+            description='Build shared libraries.')
+    variant('ssl',    default=True,
+            description="Build with OpenSSL support.")
+    variant('freetype', default='system', description='Freetype2 support',
+            values=('system', 'qt', 'none'), multi=False)
```

Comment:
Do you use `freetype=qt/none`, or have you been able to build with `freetype=system`? Ideally we could always force QT to use the Spack-built `freetype`.

I'll also note that while this translates directly to a QT configure option it is a confusing word choice for a Spack package: a "system" package in Spack often refers to a non-Spack package, where here it refers to a non-QT package that is built with Spack. The variant values `spack/qt/none` would be clearer IMO.

Reply:
I always build with `freetype=none` to reduce the chances of another conflict between a contemporary library and the 15-year-old QT4 😭 

You're right the name choice is confusing outside the context of the QT configure options. I'll make the change.